doc/Learn/Storage/Storage-GettingStarted.md

```diff
@@ -0,0 +1,116 @@
+---
+uid: Uno.Extensions.Storage.GettingStarted
+---
+
+# Getting Started with Storage Extensions
+
+> [!NOTE]
+> For an overview of the Storage feature capabilities and common file types, please refer to the [Storage Overview](xref:Uno.Extensions.Storage.Overview) page.
+
+## Prerequisites
+
+[!INCLUDE [single-project](../includes/single-project.md)]
+
+[!INCLUDE [create-application](../includes/create-application.md)]
+
+> [!TIP]
+> In case you might already have a Uno single-head project, you can still proceed from here to add the Storage feature to your existing project.
+
+So let's get started with a Step-by-Step guide to add the Storage feature to your project!
+
+## 1. Setting up your project
+
+First of all, to set up your project using the Storage feature, you need to follow these steps:
+
+1. `Storage` is provided as an Uno Feature. To enable `Storage` support in your application, add `Storage` to the `<UnoFeatures>` property in your .csproj file.
+
+    ```diff
+    <UnoFeatures>
+    +    Storage;
+    </UnoFeatures>
+    ```
+
+1. Add `.UseStorage()` in your `App.xaml.cs` HostBuilder
+
+    ```diff
+    protected async override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            // Add navigation support for toolkit controls such as TabBar and NavigationView
+            .UseToolkitNavigation()
+            .Configure(host => host
+    +           .UseStorage()
+                )
+            );
+            MainWindow = builder.Window;
+
+    #if DEBUG
+            MainWindow.UseStudio();
+    #endif
+            MainWindow.SetWindowIcon();
+            Host = await builder.Build(); // or use `Host = await builder.NavigateAsync<Shell>();` if you want to use Uno.Extensions.Navigation
+    ```
```

Comment:
Keep the code as minimal as possible when showing what needs to be added. Refer to https://github.com/unoplatform/uno.extensions/blob/9da9f0f4f830895b015dedafcb38aea5e06982ac/doc/Learn/Authentication/HowTo-MsalAuthentication.md?plain=1#L38-L50